### PR TITLE
Added mising props in Input and Toggle components types

### PR DESCRIPTION
**Pull request by MutterPedro**
## Description

- Added `onChange` prop to `Toggle` component types;
- Added `minLength` and `maxLength` props to `Input` componet types;

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] IE 11
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile
- [ ] Ux review validation

### CODE REVIEW

**Comment by rapahaeru** [comment]
In `Form.unit.test.jsx`, the props minLength and maxLength are passing a string value, instead number that you setted.

image

image

File: components/Toggle/index.d.ts [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
 export interface ToggleProps {
-    id?: string;
-    checked?: boolean;
-    theme?: {
-        colors?: object;
-        spacing?: object;
-        baseFontSize?: number;
-    };
+  onChange?: React.ChangeEventHandler<HTMLInputElement>;
```
Comment (rapahaeru) [inline components/Toggle/index.d.ts]: i think this prop `onChange` should be in the `Toggle` component too.
I noticed that component prop only stays here.
Or i missing something?

Comment (MutterPedro) [reply components/Toggle/index.d.ts]: Do you mean to add it to the Proptypes too?

**Pull request merged**